.jenkins/cscs-ault/batch.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash -l
+
+# Copyright (c) 2020 ETH Zurich
+#
+# SPDX-License-Identifier: BSL-1.0
+# Distributed under the Boost Software License, Version 1.0. (See accompanying
+# file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+set -eux
+
+# Make the name of the directories unique otherwise directories are removed by
+# another job
+orig_src_dir="$(pwd)"
+pika_dir="/dev/shm/pika"
+src_dir="${pika_dir}/src_${job_name}"
+build_dir="${pika_dir}/build_${job_name}"
+install_dir="${pika_dir}/install_${job_name}"
+
+# Clean up directories older than 7 days, find fails if dir does not exist
+test -d ${pika_dir} && find ${pika_dir}/* -type d -ctime +4 -exec rm -rf {} \; 2> /dev/null
```

Comment:
This also fails if `pika_dir` exists but is empty. Do you still need this at all? If you do:
```suggestion
test -d ${pika_dir} && find ${pika_dir} -type d -ctime +4 -exec rm -rf {} \; 2> /dev/null
```

Reply:
`find ${pika_dir}` also prints pika_dir, but if I do `find ${pika_dir} -mindepth 1`, it should work

Reply:
Thanks for noticing!